pkg/k8s/apis/cilium.io/v2/validator/validator_test.go

```diff
@@ -20,11 +20,16 @@ import (
 	"encoding/json"
 	"testing"
 
+	"github.com/sirupsen/logrus"
 	. "gopkg.in/check.v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"sigs.k8s.io/yaml"
 )
 
+var (
+	log = logrus.New()
```

Comment:
I think we have a convention for instantiating the logger, see other packages.

Reply:
(Actually, ignore this comment if superseded by the dependency-injection question above.)